spec/support/webmock/court_data_api/webmock.rb

```diff
@@ -223,13 +223,9 @@
     )
   end
 
-  # This should not happen, it should not be needed
-  # we should not calling CDAPI for hearing data
   config.before(:each, :stub_v2_hearing_data) do
     stub_request(
-      :get, %r{/v2/hearings/#{hearing_id}\?}
-    ).with(
-      query: { date: '2019-10-23' }
+      :get, %r{/api/internal/v2/hearing_results/#{hearing_id}}
```

Comment:
something looks wrong to me here:
we are linking the
`/api/internal/v2/hearing_results/`, which is a CDA call
with the file
`cd_api/hearing_data_response.json`

Reply:
Yes, you probably want to move the file from `cd_api/hearing_data_response.json` to `cda/hearing_data_response.json`, good point.